R/model_dashboard.R

```diff
@@ -105,3 +105,8 @@ model_dashboard <- function(model,
     utils::browseURL(report_path)
   }
 }
+
+# necessary for writing a mock test (see `/tests/testthat/test-model-dashboard.R`)
+# cf. https://testthat.r-lib.org/reference/local_mocked_bindings.html#namespaced-calls
+browse_url_wrapper <- function(...) utils::browseURL(...)
+browse_mock_function <- function() browse_url_wrapper()
```

Comment:
I'm not familiar with mocking but it looks like the test passes when I use `browse_url_wrapper()`, making `browse_mock_function()` useless